filetree/node.go

```diff
@@ -314,30 +321,35 @@ func (fn *Node) dirFileList() []fs.FileInfo {
 			files = append(files, info)
 		}
 	}
-	doModSort := fn.FileRoot.SortByModTime
+	doModSort := fn.FileRoot().SortByModTime
 	if doModSort {
-		doModSort = !fn.FileRoot.dirSortByName(core.Filename(path))
+		doModSort = !fn.FileRoot().dirSortByName(core.Filename(path))
 	} else {
-		doModSort = fn.FileRoot.dirSortByModTime(core.Filename(path))
+		doModSort = fn.FileRoot().dirSortByModTime(core.Filename(path))
 	}
 
-	if fn.FileRoot.DirsOnTop {
+	if fn.FileRoot().DirsOnTop {
 		if doModSort {
-			sortByModTime(dirs)
-			sortByModTime(files)
+			sortByModTime(dirs, isFS) // note: FS = ascending, otherwise descending
```

Comment:
Why?

Reply:
this is for tensorfs -- should add a more general option later but not now.

Reply:
there are a LOT of addtional things that need to be updated here for tensorfs..